### PR TITLE
Landing Page

### PR DESCRIPTION
**Pull request by simrin051**
Files To Check

1. index.html
2. style.css

iPhone SE Mobile view

image

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        <div class="leftPanel">
+            <h2>Build Your UI Faster</h2>
+            <p>Copy and paste the components at one click. It is user-friendly and customizable. Spend less time in designing and concentrate more on logic without hustling on UI.</p>
+            <div class="buttons">
+                <button name="submit" class="action_btn installation" type="submit" value="installation"><a href="#installation">Installation</a></button>
+                <button name="submit" class="action_btn documentation" type="submit" value="documentation"><a href="./docs/docs.html">Documentation</a></button>
+            </div>
+        </div>
+        <div class="rightPanel">
+            <img src="./assets/floral.svg" class="floral">
```
Comment (purvasharmadev) [inline index.html]: add alt attributes  in img tags

File: style.css [hunk 2]
```diff
@@ -0,0 +1,295 @@
...
+    display: flex;
+    justify-content: end;
+    right: 0px;
+    float: right;
+}
+
+.panel {
+    display: flex;
+    flex-direction: row;
+
```
Comment (purvasharmadev) [inline style.css:43]: Suggestion: remove extra spaces between code or add comments to separate them.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,295 @@
...
+    color: var(--primary-color);
+}
+
+.sidenav {
+    display: none;
+}
+
+@media only screen and (max-width: 1024px) {
+    body {
+        width: 100vw !important;
```
Comment (purvasharmadev) [inline style.css]: Avoid using !important,  handle it using source order.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,295 @@
...
+    }
+
+    .header-links {
+        display: none;
+    }
+
+    .sidenav .closebtn {
+        position: absolute;
+        top: 0;
+        right: 25px;
```
Comment (purvasharmadev) [inline style.css]: use rems or ems instead of px.

**Review by purvasharmadev — CHANGES_REQUESTED** [review]
Fix: 
- Avoid !important 
- use rems or ems for value more than 10-12px
- Add alt attribute to img tags

Suggestion:
- Avoid extra lines and spaces
- Comment your code.

Appreciation:
- The UI is looking amazing.
- code is clean.
- Descriptive and small PR.

**Review by abhisheksahani-nova — CHANGES_REQUESTED** [review]
Make the changes suggested by Purva , then it will be good to merge.

File: style.css (hunk 2 above)
Comment (simrin051) [reply style.css:43]: Sudesh had asked to follow this way for easy understanding of css classes applied to an element.

**Pull request merged**